config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
 
+  resources :legal_forms
```

Comment:
I would group this together with the other resources, so the routes file doesn't get so fragmented.